### PR TITLE
Added Django's template standard

### PR DESCRIPTION
**Pull request by ar4s**
New standard in new Ralph.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,4 +57,30 @@ Feel free to explore beauty of Python programming. Make contributions that matte
 
 The answer to the question "When is it ready?" is "When it is ready" :-)
 
+## Django's template standard
 
+There are few simple rules:
+
+1. Blank line between ``load`` and rest of template.
+2. Blank line between block section.
+3. If you open some HTML tag or Django template tag you must indent line, exclude simple code (see example).
```
Comment (mkurek) [inline CONTRIBUTING.md]: excluding

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,4 +57,30 @@ Feel free to explore beauty of Python programming. Make contributions that matte
 
 The answer to the question "When is it ready?" is "When it is ready" :-)
 
+## Django's template standard
 
+There are few simple rules:
+
+1. Blank line between ``load`` and rest of template.
```
Comment (vi4m) [inline CONTRIBUTING.md]: the rest of a template

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -57,4 +57,30 @@ Feel free to explore beauty of Python programming. Make contributions that matte
...
+2. Blank line between block section.
```
Comment (vi4m) [inline CONTRIBUTING.md]: a block section

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -57,4 +57,30 @@ Feel free to explore beauty of Python programming. Make contributions that matte
...
+3. If you open some HTML tag or Django template tag you must indent line, exclude simple code (see example).
+
+All of this is for human readability not for browsers.
```
Comment (vi4m) [inline CONTRIBUTING.md]: We prefer keeping **source** template clean, not bothering with the output html indentation, which could be unreadable.

**Comment by xliiv** [comment]
Django uses: http://editorconfig.org/
It won't cover all our cases, but maybe we should use it?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -57,4 +57,33 @@ Feel free to explore beauty of Python programming. Make contributions that matte
 
 The answer to the question "When is it ready?" is "When it is ready" :-)
 
+## Django's template standard
 
+There are few simple rules:
+
+1. Blank line between ``load`` and rest of a template.
+2. Blank line between a block section.
```
Comment (ghost) [inline CONTRIBUTING.md]: `Blank line after a block section.`
or
`Blank line between a block sections.`

**Comment by mkurek** [comment]
:ok:

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -57,4 +57,33 @@ Feel free to explore beauty of Python programming. Make contributions that matte
 
 The answer to the question "When is it ready?" is "When it is ready" :-)
 
+## Django's template standard
 
+There are few simple rules:
+
+1. Blank line between ``load`` and the rest of a template.
+2. Blank line between a block sections.
```
Comment (damian1976) [inline CONTRIBUTING.md]: without "a"

**Comment by ghost** [comment]
:ok:

**Comment by vi4m** [comment]
It is **not** allowed to introduce new syntax rules without providing automated checker. So this PR is not valid. It should be easy to write one even basic and add to the Makefile.

**Comment by vi4m** [comment]
Please create new issue and connect to this one because we can't unmerge it.

**Pull request merged**